packages/build/src/plugins_core/functions/zisi.js

```diff
@@ -20,8 +20,10 @@ export const getZisiParameters = ({
     normalizeFunctionConfig({ buildDir, featureFlags, functionConfig: object, isRunningLocally, nodeVersion }),
   ])
   const zisiFeatureFlags = getZisiFeatureFlags(featureFlags)
+  // Only internal functions are allowed to have a json config file
+  const configFileDirectories = [resolve(buildDir, '.netlify', 'functions-internal')]
```

Comment:
We have a constant for this: https://github.com/netlify/build/blob/main/packages/build/src/core/constants.js#L46